pkg/monitor/monitorapi/identification.go

```diff
@@ -53,23 +45,32 @@ func IsNode(locator string) bool {
 }
 
 func NodeFromLocator(locator string) (string, bool) {
-	if !strings.HasPrefix(locator, "node/") {
```

Comment:
this made us order sensitive and this https://github.com/openshift/machine-config-operator/pull/3767 made the namespace get specified.